examples/panel_app/README.md

```diff
@@ -0,0 +1,3 @@
+
+
+ragna api -c demo & python examples/panel_app/main.py && fg
```

Comment:
I've added a CLI for the UI, which automatically starts the API if needed. Can we remove `examples/panel_app`?